android/res/values-nl/strings.xml

```diff
@@ -598,6 +598,7 @@
 	<string name="bookmarks_error_title_list_name_already_taken">Deze naam is al in gebruik</string>
 	<string name="bookmarks_error_message_list_name_already_taken">Kies alstublieft een andere naam</string>
 	<string name="please_wait">Even geduld aub…</string>
+	<string name="phone_number">Telefoonnummer</string>
```

Comment:
Did you manually add it? I see that these strings were already present in the master branch, in data/strings/strings.txt and in XML android resources.

Reply:
Viacheslav Greshilov removed "phone_number" string in commit [#cca70039](https://github.com/organicmaps/organicmaps/commit/cca7003972cd540c6f1c7649250eb08fe5abc775) and generated strings.xml in commit [#2b46f73d](https://github.com/organicmaps/organicmaps/commit/2b46f73d3a8b65cfafe9383eb77bdf97a233f7a9)
I restored "phone_number" in strings.xml, but looks like I need to revert data/strings/strings.txt first

Reply:
Wow, this string was unused, interesting. Thanks!